docs/contributing.md

```diff
@@ -189,6 +194,88 @@ occasionally check the terminal from which you ran `mkdocs serve` for
 any build errors or warnings. Warnings about
 `"GET /versions.json HTTP/1.1" code 404` are expected and harmless.
 
+### How to build the entire website (not usually necessary)
+
+The full `jj` website includes the documentation for several `jj` versions
+(`prerelease`, latest release, and the older releases). The top-level
+URL <https://martinvonz.github.io/jj> redirects to
+<https://martinvonz.github.io/jj/latest>, which in turn redirects to
+the docs for the last stable version.
+
+The different versions of documentation are managed and deployed with
+[`mike`](https://github.com/jimporter/mike), which can be run with
+`poetry run -- mike`.
+
+On a POSIX system or WSL, one way to build the entire website is as follows (on
+Windows, you'll need to understand and adapt the shell script):
+
+1. Check out `jj` as a co-located `jj + git` repository (`jj clone --colocate`),
+cloned from your fork of `jj` (e.g. `jjfan.github.com/jj`). You can also use a
+pure Git repo if you prefer.
+
+2. Make sure `jjfan.github.com/jj` includes the `gh-pages` branch of the jj repo
+and run `git fetch origin gh-pages`.
+
+3. Go to the Github repository settings, enable Github Pages, and configure them
```

Comment:
nit: Github->GitHub (uppercase H) throughout

Reply:
I added a commit to make this fix everywhere.